plugins/pulp_rpm/plugins/distributors/yum/publish.py

```diff
@@ -37,6 +37,15 @@
 logger = util.getLogger(__name__)
 
 
+def make_packages_relative_path(filename):
```

Comment:
Since we are not making any changes to DRPMs  this function is used only once here while you still generate this path in some other modules. Maybe it is better to put it in some RPM plugin util module and use it anywhere it's needed.
In case we will change layout again we will need to fix it in one place.
What do you think?